extension/src/popup/components/sendPayment/styles.scss

```diff
@@ -1,4 +1,26 @@
-// TODO - separate style sheet
```

Comment:
I kept the sendPayment styling in one style sheet. imo it made more sense because of the shared classes ( eg. .btn-continue and .header). Open to discussing separating though

Reply:
A convention i've been following in this repo is to create `basics` for shared things.

So in this case, you might make a folder called `basics/SendTransaction` (assuming these buttons/headers will be reused when we want to Sign/Submit other transactions)

You can take a look at `basics/Forms` as a template. You would probably export something like `export const BtnContinue = () => <div className="SendTransaction__btn-continue`>{children"</div>`.

The danger of just having a class called `btn-continue` rather than something like `SendTransaction__btn-continue` is that you could run into a naming collision.  You can imagine `btn-continue` is  a name that would come up a lot bc we're not very creative thinkers when it comes to class naming lol

Reply:
good point on the naming collisions, I changed it to `SendPayment__btn-continue` and `SendPayment__header`. https://github.com/stellar/freighter/pull/407/commits/557f4beef936ce430302c92773834467ebb0a96f

I don't think these should live in `basics` though, I left them in `sendPayment` folder. They're pretty specific to this send payment flow (eg. the header I don't see being used elsewhere and the btn-continue is just a space wrapper)

lmk what you think

Reply:
Ok cool, then then I'm fine with this!